prime-router/src/main/kotlin/history/azure/HistoryDatabaseAccess.kt

```diff
@@ -136,12 +139,22 @@ abstract class HistoryDatabaseAccess(
     private fun createWhereCondition(
         organization: String,
         orgService: String?,
+        reportId: UUID?,
+        fileName: String?,
         since: OffsetDateTime?,
         until: OffsetDateTime?,
         showFailed: Boolean,
     ): Condition {
         var filter = this.organizationFilter(organization, orgService)
 
+        if (reportId != null) {
```

Comment:
Optional, but this does still feel weird since these are mutually exclusive?  I could see up dating the HTTP function to check the parameters and return a `422` if both are provided.

Otherwise, I just think we need really good documentation on what the outcome would be if you provide both

Reply:
Agree, should this be checked in the ReportFileFunction.kt -> getListByOrg() method or here?

Reply:
My two cents would be close to the Function itself to make it easier to generate the correct HTTP response